packages/dispatcher/src/index.ts

```diff
@@ -12,12 +14,15 @@ import logger from "./logger";
 
 export class SlackDispatcher {
   private app: App;
-  private jobManager: KubernetesJobManager;
+  private jobManager?: KubernetesJobManager;
```

Comment:
why do you need the jobManager, isn't it handled in the orchestrator? if yes, delete it from here.